plant-crossing/screens/GardenScreen.tsx

```diff
@@ -57,7 +59,8 @@ export default function GardenScreen() {
   return (
     <View style={styles.container}>
       <Text style={styles.title}>Garden Page</Text>
-      <Text style={styles.gardenSection}>placeholder</Text>
+      {/* <Text style={styles.gardenSection}>placeholder</Text> */}
+      <GardenGrid></GardenGrid> 
```

Comment:
is it possible to change the style to have flex:3 so that the grid will take up 3/4s of the page? i think that edit would either be here or in GardenPlots.tsx where you create the flatlist